Microsoft.Toolkit.Uwp.UI.Controls/BladeView/BladeView.xaml

```diff
@@ -23,6 +23,7 @@
         <Setter Property="ScrollViewer.IsDeferredScrollingEnabled" Value="False" />
         <Setter Property="ScrollViewer.BringIntoViewOnFocusChange" Value="True" />
         <Setter Property="UseSystemFocusVisuals" Value="True" />
+        <Setter Property="AutomationProperties.Name" Value="{Binding BladeStatus}" />
```

Comment:
For this and the Expand/Collapse here and above; we should be modifying the button's name itself to indicate what will happen when the user toggles it.  (For 6.0, we may want to make this a toggle button instead anyway.)

Reply:
Are you saying that instead of setting a status and changing the AutomationProperty.Name (this is binded to a property modified on an event handler in BladeItem.cs .) , we change the button Name property directly instead?

Reply:
Yup! :)

Reply:
But I think we can't change the button name, as in the BladeItem.cs it uses the name string in the get template child to set the event handler .

Reply:
The x:Name used to grab the control and the AutomationProperties.Name are different, they shouldn't effect one another.

Reply:
I think we were on different pages, could you check if the last commits is closer to what you meant?